### DISCUSSION TITLE
Create exe release of auto-py-to-exe

### DISCUSSION THREAD

**Discussion by tap6** [brentvollebregt/auto-py-to-exe#285] [General]
咱就是说python源码的环境是真复杂，就是有源码，下载下来用也很复杂。。不如直接创建一个exe，感觉大部分人不会去看源码实现的，所以哥们麻烦以后发布release的时候附带个exe？

**Comment by brentvollebregt** [comment]
Have you read over any of the issues relating to this? In #267 and #239 I note this is not possible.

Pretty much if this is packaged into a single executable, then the Python packaged within the exe is what's searched for when looking for modules. This means if you have something like numpy installed locally, it will not be picked up as the internal Python distribution will be searched.

You can get around this by telling PyInstaller where to look for other modules but if you are really wanting to have an exe to make things easier - I have a feeling you may not be able to figure this out easily. Comparing the struggle for getting the correct paths for the libraries you want somewhere else and just downloading the package - it's a no-brainer.

Also, packaging auto-py-to-exe will mean it's locked at a specific version. It won't care if you have Python version X installed - it's always going to use the version Y packaged.
